test/e2e/conftest.py

```diff
@@ -173,28 +173,32 @@ async def delete_old_org_accounts() -> None:
             continue
 
 
-async def create_organization_account() -> Account:
+async def get_or_create_test_organization_account() -> Account:
     config = await ConfigManager().refresh()
     client = LayerClient(config.client, logger)
-    org_account_name, display_name = pseudo_random_account_name()
-    account = client.account.create_organization_account(
-        org_account_name, display_name, deletion_allowed=True
-    )
 
-    # We need a new token with permissions for the new org account
-    # TODO LAY-3583 LAY-3652
-    await ConfigManager().refresh(force=True)
+    try:
+        org_account = client.account.get_account_by_name(
+            PERMANENT_TEST_ORG_ACCOUNT_NAME
+        )
+    except LayerClientResourceNotFoundException:
+        org_account_name, display_name = pseudo_random_account_name()
+        org_account = client.account.create_organization_account(
+            org_account_name, display_name, deletion_allowed=True
+        )
```

Comment:
What is the purpose of the fallback? If the org is not there, but we should probably just fail hard

Reply:
the purpose is to allow us to merge this before we have the test account created on all stacks

Reply:
well I should have put a better description first :) thanks!